components/sup/src/event/nats_message_stream.rs

```diff
@@ -33,64 +119,22 @@ impl NatsMessage {
 pub struct NatsMessageStream(pub(super) UnboundedSender<NatsMessage>);
 
 impl NatsMessageStream {
-    pub async fn new(supervisor_id: &str, config: EventStreamConfig) -> Result<NatsMessageStream> {
-        let EventStreamConfig { url,
-                                token,
-                                connect_method,
-                                server_certificate,
-                                .. } = config;
-
-        let mut client = Client::new(vec![url]);
-
-        // Configure the client connect message
-        client.connect_mut()
-              .await
-              .name(format!("hab_client_{}", supervisor_id))
-              .verbose(true)
-              .token(token.to_string());
-
-        // Configure the tls connector
-        let mut tls_connector = TlsConnector::builder();
-        for certificate in habitat_core::tls::native_tls_wrapper::certificates(None)? {
-            tls_connector.add_root_certificate(certificate);
-        }
-        if let Some(certificate) = server_certificate {
-            tls_connector.add_root_certificate(certificate.into());
-        }
-        let tls_connector = tls_connector.build()?;
-        client.set_tls_config(tls_connector).await;
-
+    pub async fn new(supervisor_id: String, config: EventStreamConfig) -> Result<NatsMessageStream> {
```

Comment:
event is passing a `&str` so I don't think this compiles.

Reply:
My bad.  Missed one part when I reverted some experimental changes before I committed the code.  Will revert on next commit.